services/frontend-v3/src/components/admin/userTable/UserTableView.jsx

```diff
@@ -121,17 +119,29 @@ const UserTableView = ({
         setTimeout(() => searchInput.select());
       }
     },
-    render: (text) =>
-      searchedColumn === dataIndex ? (
-        <Highlighter
-          highlightStyle={{ backgroundColor: "#ffc069", padding: 0 }}
-          searchWords={[searchText]}
-          autoEscape
-          textToHighlight={text.toString()}
-        />
-      ) : (
-        text
-      ),
+    render: (text, record) => {
+      const view =
+        searchedColumn === dataIndex ? (
+          <Highlighter
+            highlightStyle={{ backgroundColor: "#ffc069", padding: 0 }}
+            searchWords={[searchText]}
+            autoEscape
+            textToHighlight={text.toString()}
+          />
+        ) : (
+          text
+        );
+
+      if (linkKey && record[linkKey]) {
+        return (
+          <Link to={record[linkKey]} target="_blank">
```

Comment:
I dont know if we should open the link in a new tab. The standard is if you are opening an external link it should be opened a new tab but if it is internal it should be on the same tab. The idea is the user would redirect to that user in the same window instead of having to switch between tabs. What do you think?

Reply:
Makes sense to keep them inside the same tab, will remove the target